examples/custom_paintable/custom_paintable/imp.rs

```diff
@@ -0,0 +1,81 @@
+use gtk::prelude::*;
+use gtk::subclass::prelude::*;
+use gtk::{gdk, glib, graphene, gsk};
+#[derive(Default)]
```

Comment:
Empty line between after the imports
```suggestion

#[derive(Default)]
```